codespell_lib/data/dictionary.txt

```diff
@@ -18112,8 +18112,15 @@ scandanavia->Scandinavia
 scaned->scanned
 scaning->scanning
 scannning->scanning
-scarch->search
+scarch->search, scorch,
 scaricity->scarcity
+scatch->sketch, catch,
```

Comment:
Suggest scratch to all these too, I think first in the list.

I think for this one the suggestions order should be:
scatch->catch, scratch, sketch,

And equivalent for the others.

Reply:
e0eee9c